### PR TITLE
reorganise gsoc22

### PR DESCRIPTION
**Pull request by Lovkush-A**
the changes include:

- adding a list of contents at the top
- combining 'how to apply' and 'selection process' into one section
- combining 'what we expect from you' and 'what we are looking for' into one section
- adding a timeline to the application process
- convert some paragraphs into bullet lists
- various minor tweaks to the wording

### CODE REVIEW

**Comment by Lovkush-A** [comment]
@fkiraly. I dont seem to be able to add reviewers. Can you have a quick scan to check for any glaring errors?

**Comment by fkiraly** [comment]
Yes - I´ve now added the core developer team as maintainers.

File: internships/gsoc2022.md
Comment (fkiraly) [inline internships/gsoc2022.md:26]: mentions "two main phases", but then lists three items. Confusing.
Timeline afterwards does not map clearly onto the phases.

File: internships/gsoc2022.md
Comment (fkiraly) [inline internships/gsoc2022.md:28]: missing: entrance task (PR), condition for the sktime application. Easily confusing.

File: internships/gsoc2022.md
Comment (fkiraly) [inline internships/gsoc2022.md:38]: confusing - three steps vs two phases.

File: internships/gsoc2022.md
Comment (fkiraly) [inline internships/gsoc2022.md:61]: suggest, add the recommendation to come prepared for screen sharing, or with a link to a repo

File: internships/gsoc2022.md
Comment (fkiraly) [inline internships/gsoc2022.md:63]: there will also be technical questions

File: internships/gsoc2022.md
Comment (fkiraly) [inline internships/gsoc2022.md:95]: perhaps add, weekly individual mentoring meeting, max two working day latency in reaction from core devs, daily standups with the community

File: internships/gsoc2022.md
Comment (fkiraly) [inline internships/gsoc2022.md:107]: 2022, leave that in - the list changes every year.

**Review by fkiraly — CHANGES_REQUESTED** [review]
Thanks, great work, definitely an improvement.

My main blocking comment is on the "two phases" vs "three steps" and obscurity around the requirements to complete the application. Added  details on that, and additional other comments, above.

File: internships/gsoc2022.md
Comment (Lovkush-A) [reply internships/gsoc2022.md:95]: done

File: internships/gsoc2022.md
Comment (Lovkush-A) [reply internships/gsoc2022.md:63]: clarified final phrase to make this clear

File: internships/gsoc2022.md
Comment (Lovkush-A) [reply internships/gsoc2022.md:28]: adding entrance task in the relevant places

**Comment by Lovkush-A** [comment]
@fkiraly I have made changes according to all the things you suggested. Ready for a second review!

**Review by fkiraly — APPROVED** [review]
looks good, thanks!

**Pull request merged**
